### DISCUSSION TITLE
FcRn nonbinding compound

### DISCUSSION THREAD

**Discussion by kajasir** [Open-Systems-Pharmacology/Forum#1727] [Questions & Problems] [answered]
Hello,
this is probably very basic question but I would like to be sure. If I have a FcRn nonbinding compound how do I set it that the compound will not be bound to FcRn?
Thank you for any help

**Comment by Christoph27** [comment, marked as answer]
The degree of binding to FcRn is set via Kd values in the "Advanced Parameters" tab of the compound, see screenshot. The default in PK-Sim are high Kd-values corresponding to no binding, so in your case you can keep the default. 

<img width="257" alt="image" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/26439150/649de9de-c03e-4be2-8ae6-a7a1a9b043b2">
